setup-ovs.sh

```diff
@@ -2,22 +2,38 @@
 
 set -ex
 
+rm -rf /etc/NetworkManager/system-connections/*
```

Comment:
What about the extra -merged directories? Don't we want to clean up their content too?

Reply:
I think that if we follow the configurable path set on `nmcli --print-config` then we should always point to the folder where the connections are stored ([example](https://github.com/coreos/coreos-assembler/pull/2367/files#diff-d3d2a708b2f5727515ca091586b987a845a198307fc62584ce5047654ddfdfafR286-R290))

Reply:
We are deleting all the subfolders under system-connections so we are ok here and we are cleaning the relevant sub directory before we are coping the new configuration.